tmpl/tmpl.go

```diff
@@ -87,6 +89,12 @@ func Parse(s string, t time.Time) string {
 	hour := fmt.Sprintf("%02d", t.Hour())
 	s = regHour.ReplaceAllString(s, hour)
 
+	// {HOST}
+	if h, err := os.Hostname(); err == nil {
+		s = strings.Replace(s, "{HOST}", h, -1)
+		// should we handle error case?
```

Comment:
@jbsmith7741 should we return / report an error here or just return the unmodified HOST slug as I've done here

Reply:
Let's keep what you have. The method doesn't have a return time and changing it would break the api.  We may want to support both upper and lower case options {host} and {HOST}